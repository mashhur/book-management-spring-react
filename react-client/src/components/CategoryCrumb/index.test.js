const { shallow } = require("enzyme");
const { default: CategoryCrumb } = require(".");
const { checkProp } = require("../../tests/testUtils");

const defaultProps = {
	category: "Horror",
};
it("should match the default snapshot", () => {
	const wrapper = shallow(<CategoryCrumb />);
	expect(wrapper).toMatchSnapshot();
});
it("doesnot throw warning with expected props", async () => {
	checkProp(CategoryCrumb, defaultProps);
});
