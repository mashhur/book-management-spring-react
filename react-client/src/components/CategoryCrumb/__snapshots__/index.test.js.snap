// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the default loading snapshot 1`] = `ShallowWrapper {}`;

exports[`should match the default snapshot 1`] = `ShallowWrapper {}`;
